frontend/src/theme/selectStyle.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from "./colors";
+
+const optionBackgroundColor = (isSelected, isFocused) => {
```

Comment:
seeing this error when I compile:
```
/app/src/theme/selectStyle.tsx
TypeScript error in /app/src/theme/selectStyle.tsx(3,32):
Parameter 'isSelected' implicitly has an 'any' type.  TS7006

    1 | import colors from "./colors";
    2 | 
  > 3 | const optionBackgroundColor = (isSelected, isFocused) => {
      |                                ^
    4 |   if (isSelected) {
    5 |     return colors.teal[100];
    6 |   }
```

Reply:
setting their types to boolean doesn't fix this :(
```
/app/src/theme/selectStyle.tsx
TypeScript error in /app/src/theme/selectStyle.tsx(3,32):
Parameter 'isSelected' implicitly has an 'any' type.  TS7006

    1 | import colors from "./colors";
    2 | 
  > 3 | const optionBackgroundColor = (isSelected: boolean, isFocused: boolean) => {
      |                                ^
    4 |   if (isSelected) {
    5 |     return colors.teal[100];
    6 |   }
```

Reply:
Mine works fine with this change, but then again it also worked when no type is specified

Reply:
@Safewaan @phamkelly17  Getting the same error in compilation:

<img width="1274" alt="Screen Shot 2023-06-19 at 8 47 28 PM" src="https://github.com/uwblueprint/supportive-housing/assets/53386475/1e6c9123-2c7f-4b4b-955c-9ab1b844f57a">

Reply:
@kevin-pierce could you try adding the change safwaan tried two comments above because that wasn't working for him either

Reply:
Yeah it worked locally, though another issue arose after implementing that fix (don't know if I'm missing some step in branch setup):
<img width="733" alt="Screen Shot 2023-06-19 at 8 54 46 PM" src="https://github.com/uwblueprint/supportive-housing/assets/53386475/fee41bcd-18e7-4011-8c36-45717aef3820">

Reply:
no I added that T_T... could you just try changing that to `any` type

Reply:
🎉 

<img width="623" alt="Screen Shot 2023-06-19 at 9 00 04 PM" src="https://github.com/uwblueprint/supportive-housing/assets/53386475/e1097887-1cad-4b58-ad51-9f7d2b4426a7">